sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 1.2.1 (Unreleased)
+## 1.2.1 (2020-12-08)
 
+- Support custom auth scopes [PR 12752](https://github.com/Azure/azure-sdk-for-js/pull/12752)
```

Comment:
Is there a way to explain this better? For new users reading this, what does "auth scopes" mean?

Reply:
Updated the description